edgegraph/traversal/breadthfirst.py

```diff
@@ -110,16 +110,16 @@ def bfs(
     return None
 
 
-def bft(
+def ibft(
     uni: Universe,
     start: Vertex,
     direction_sensitive: int = helpers.DIR_SENS_FORWARD,
     unknown_handling: int = helpers.LNK_UNKNOWN_ERROR,
     ff_via: Callable | None = None,
     ff_result: Callable | None = None,
```

Comment:
Should the `ff_result` argument be removed from this function and instead applied by the user?  Because this now returns an iterator, the following would be exactly equivalent:

```python
trav = ibft(uni, start, ff_result)
trav = filter(ff_result, ibft(uni, start))
```

Not certain if any performance differences would be notable, so need to test; as well as determine which approach is preferable.

**NOTE**: If this is removed, the generator and non-generator function signatures would differ, which means documentation needs updating for both.

Reply:
***Nah.***